pkg/instancetype/upgrade/upgrade_test.go

```diff
@@ -64,6 +71,22 @@ var _ = Describe("ControllerRevision upgrades", func() {
 		controllerrevisionInformerStore cache.Store
 	)
 
+	sanityUpgrade := func(vm *virtv1.VirtualMachine) error {
+		stores := []cache.Store{controllerrevisionInformerStore}
+		var listOfObjects [][]interface{}
+
+		for _, store := range stores {
+			listOfObjects = append(listOfObjects, deepCopyList(store.List()))
+		}
+
+		err := upgradeHandler.Upgrade(vm)
```

Comment:
All callers assert `err` to be `nil` at the moment (we should really add more coverage) so we may as well do that here until there's a need to assert something else and return `err`.

Reply:
Let me kindly disagree :) 
From my point of view the `sanityExecute` should only ensure the cache is not corrupted, deferring any other expectation to the caller.
If you have a strong opinion, I will change it :) Please, let me know. Thanks

Reply:
I don't think the naming suggests that specific usage tbh and in cases where all callers are doing something with the returned results I'd expect us to at least for now fold it into the function. That said this isn't a hill to die on and the coverage needs to be extended anyway to cover cases where we need to assert `err` as something other than `nil`.